autoreject/utils.py

```diff
@@ -377,14 +378,14 @@ def get_picks_bad_good(info, picks_meg):
     # as the MNE interpolation code is not fogriving.
     # This is why we picked the info.
     mapping = _fast_map_meg_channels(
-        picked_info.copy(), pick_from=picks_good, pick_to=picks_bad,
+        picked_info, pick_from=picks_good, pick_to=picks_bad,
         mode=mode)
     # the downside is that the mapping matrix now does not match
     # the unpicked info of the data.
     # Since we may have picked the info, we need to double map
     # the indices.
     _, picks_good_, picks_bad_orig = get_picks_bad_good(
-        inst.info.copy(), picks)
+        inst.info, picks)
```

Comment:
@dengemann turns out these are not needed. I went over carefully to check if they are not being accidentally modified inside the function. I think we were extra careful when coding them but it's hurting performance now